### PR TITLE
Show program rules in nanite cloud without having to have sensor disks inserted

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

Currently nanite cloud controllers will only display the rules for any given program when there is a disk inserted with a can_rule program stored on it.

This PR adds a little bit of additional logic to ALSO display the rules section if the program also has rules stored.

Finally, the formatting of the rules section has been made a little bit more aesthetically pleasing at the cost of taking up more vertical space.

## Current Live Behaviour

Program with rules added does not display rules because there is no disk rule-program disk inserted.
image
image

When a rules disk is inserted, it displays the rules. Formatting is ugly.
image

## New Behaviour

No rules added, no rules shown.
image

With and without rules disk inserted, rules now display properly.
image
image

New rules formatting.

## Why It's Good For The Game

image

Improving things good.

## Changelog
:cl:
tweak: Nanite Cloud Controllers now display stored rules for programs without requiring specific disks inserted.
/:cl:

### CODE REVIEW

**Review by stylemistake — DISMISSED** [review]

File: tgui/packages/tgui/interfaces/NaniteCloudControl.js [hunk 1]
```diff
@@ -253,7 +253,7 @@ export const NaniteCloudBackupDetails = (props, context) => {
                             program_id: program.id,
                             rule_id: rule.id,
                           })} />
-                        {rule.display}
+                        {` ${rule.display}`}<br />
                       </Fragment>
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/NaniteCloudControl.js:257]: Replace the whole fragment with a Box, and you won't need a br.

**Review by stylemistake — COMMENTED** [review]
Important: run `yarn install` to update deps, and rebuild tgui.

File: tgui/packages/tgui/interfaces/NaniteCloudControl.js (hunk 1 above)
Comment (Timberpoes) [reply tgui/packages/tgui/interfaces/NaniteCloudControl.js]: Done. Looks to work on testing. Thanks!

**Review by stylemistake — APPROVED** [review]

**Comment by actioninja** [comment]
Not gonna lie I didn't actually know how nanite rules worked and so didn't ever test this.

**Pull request merged**